locales/tr.strings

```diff
@@ -102,7 +102,6 @@
 "write" = "Yaz";
 "publish" = "Yayınla";
 "delete" = "Sil";
-"comments" = "Yorumlar";

```

Comment:
Why?

Reply:
> No need to delete the Comments string

Then why was it deleted here?

https://github.com/windowz414/openvk/commit/a77c7dea541e6f31eb3ad903498eae061418a08b#diff-12b70ca9f0009f215fc01a7bc214dde1bbe09335e4ee010a6ad2a8ada48fc953L116